lib/cli.js

```diff
@@ -125,6 +126,13 @@ async function translateOptions({
             rules: rule ? rule : {}
         }];
 
+        if (reportUnusedDisableDirectives || reportUnusedDisableDirectivesSeverity) {
+            if (!overrideConfig[0].linterOptions) {
+                overrideConfig[0].linterOptions = {};
+            }
+            overrideConfig[0].linterOptions.reportUnusedDisableDirectives = reportUnusedDisableDirectivesSeverity || (reportUnusedDisableDirectives ? "error" : void 0);
```

Comment:
```suggestion
            overrideConfig[0].linterOptions = {
                reportUnusedDisableDirectives: reportUnusedDisableDirectivesSeverity || "error"
            };
```

I think we can simplify this.